pkg/network/events/monitor.go

```diff
@@ -35,15 +36,22 @@ func Init() error {
 	return initErr
 }
 
-// HandlerFunc is the prototype for an event handler callback for process events
-type HandlerFunc func(*model.ProcessContext)
+type ProcessEventHandler interface {
+	HandleProcessEvent(*model.ProcessContext)
```

Comment:
Just `Handle`?

Reply:
That seems pretty generic for an interface method. The function can be part of a object which does a lot of things, so `Handle` doesn't indicate much.